dll/win32/shell32/shelldesktop/CDesktopBrowser.cpp

```diff
@@ -458,6 +460,22 @@ LRESULT CDesktopBrowser::OnGetChangeNotifyServer(UINT uMsg, WPARAM wParam, LPARA
     return (LRESULT)m_hwndChangeNotifyServer;
 }
 
+extern VOID WINAPI ShowFolderOptionsDialog(UINT Page, BOOL Async);
```

Comment:
Perhaps this could be moved into some header file.

Reply:
I thought about it but I believe this is the only external place that needs to call this function.